packages/nginx/packaging

```diff
@@ -0,0 +1,16 @@
+set -e -x
+
+echo "Extracting pcre..."
+tar xzvf pcre-8.40.tar.gz
```

Comment:
I think you'll need to fix this, or restore this version to the blogs, as I bumped it to latest (8.41) in this branch